features/skipped_steps.feature

```diff
@@ -26,6 +26,14 @@ Feature: Skipped steps
     When I run cucumber-js
     Then it passes
     And the step "a skipped step" has status "skipped"
+    And it outputs the text:
+      """
+      -
+
+      1 scenario (1 skipped)
+      1 step (1 skipped)
+      <duration-stat>
+      """
```

Comment:
Rather than testing the output, I'd prefer we create a step like `And the scenario "a scenario" has status "skipped"`. You should be able to model that step after similar steps in `features/step_definitions/json_output_steps.js`

Reply:
I agree that your suggestion would be more cleaner, and it was my first approach. However, when I checked `this.lastRun.jsonOuput` (or any json output for that matter) I found that the result of the Scenario itself is not stored, only the result of individual steps. An example below

<details><summary>output</summary>
<p>

```json
{
  "errorOutput": "",
  "jsonOutput": [
    {
      "keyword": "Feature",
      "name": "a feature",
      "line": 1,
      "id": "a-feature",
      "tags": [],
      "uri": "features/skipped.feature",
      "elements": [
        {
          "id": "a-feature;a-scenario",
          "keyword": "Scenario",
          "line": 2,
          "name": "a scenario",
          "tags": [],
          "type": "scenario",
          "steps": [
            {
              "keyword": "Before",
              "hidden": true,
              "match": {
                "location": "features/support/hooks.js:3"
              },
              "result": {
                "status": "skipped"
              }
            },
            {
              "arguments": [],
              "keyword": "Given ",
              "line": 3,
              "name": "a skipped step",
              "match": {
                "location": "features/step_definitions/skipped_steps.js:3"
              },
              "result": {
                "status": "skipped"
              }
            }
          ]
        }
      ]
    }
  ],
  "output": "--\n\n1 scenario (1 skipped)\n1 step (1 skipped)\n0m00.000s\n"
}
```

</p>
</details>

I am not sure of how to proceed or if I missing something. If needed, I can take a look and add the scenario result to the json output.

Reply:
@charlierudolph

Reply:
Sorry, after seeing this I started work on making this possible by switching out parsing the json output for event protocol output. Haven't finish that yet though. I'm fine with this as is and will update the test once I complete my task.